### PR TITLE
chore: update liquity namings to thresholdusd

### PR DESCRIPTION
**Pull request by evandrosaturnino**
## Description 

- [X] - Update README with updated info
- [X] - Refactor: Update liquity namings

## Notice 

- [x] Have you checked to ensure there aren't other open Pull Request for the same update/change? 

## Pull Request Type 

- [x] Chore update (Breaking/Non-breaking Change) 

## Testing Please outline all testing steps 

Testing locally: 
1. Clone the repo locally: `git clone https://github.com/Threshold-USD/dev.git` 
2. Install the dependecies with yarn: `yarn` 
3. Run the app: `yarn start-dev-frontend`

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -355,52 +349,52 @@ This ordered list is critical for gas-efficient redemption sequences and for the
...
-Ether is recorded on an _individual_ level, but stored in _aggregate_ in a Pool. An active Trove with collateral and debt has a struct in the TroveManager that stores its ether collateral value in a uint, but its actual Ether is in the balance of the ActivePool contract.
+Collateral is recorded on an _individual_ level, but stored in _aggregate_ in a Pool. An active Vault with collateral and debt has a struct in the TroveManager that stores its ether collateral value in a uint, but its actual collateral is in the balance of the ActivePool contract.
 
-Likewise, the StabilityPool holds the total accumulated ETH gains from liquidations for all depositors.
+Likewise, the StabilityPool holds the total accumulated collateral gains from liquidations for all depositors.
 
 **Borrower Operations**
 
-| Function                     | ETH quantity                        | Path                                       |
+| Function                     | collateral quantity                        | Path                                       |
```
Comment (vzotova) [inline README.md]:
```suggestion
| Function                     | Collateral quantity                        | Path                                       |
```

File: README.md [hunk 2]
```diff
@@ -355,52 +349,52 @@ This ordered list is critical for gas-efficient redemption sequences and for the
...
+| adjustTrove: adding collateral      | msg.value                           | msg.sender->BorrowerOperations->ActivePool |
+| adjustTrove: withdrawing collateral | _collWithdrawal parameter           | ActivePool->msg.sender                     |
 | closeTrove                   | All remaining                       | ActivePool->msg.sender                     |
 | claimCollateral              | CollSurplusPool.balance[msg.sender] | CollSurplusPool->msg.sender                |
 
-**Trove Manager**
+**Vault Manager**
 
-| Function                                | ETH quantity                           | Path                          |
+| Function                                | collateral quantity                           | Path                          |
```
Comment (vzotova) [inline README.md]:
```suggestion
| Function                                | Collateral quantity                           | Path                          |
```

File: README.md [hunk 3]
```diff
@@ -410,25 +404,25 @@ Likewise, the StabilityPool holds the total accumulated ETH gains from liquidati
 | batchLiquidateTroves (redistribution).  | collateral to be redistributed         | ActivePool->DefaultPool       |
 | redeemCollateral                        | collateral to be swapped with redeemer | ActivePool->msg.sender        |
 | redeemCollateral                        | redemption fee                         | ActivePool->PCV       |
-| redeemCollateral                        | trove's collateral surplus             | ActivePool->CollSurplusPool |
+| redeemCollateral                        | vault's collateral surplus             | ActivePool->CollSurplusPool |
 
 **Stability Pool**
 
-| Function               | ETH quantity                     | Path                                              |
+| Function               | collateral quantity                     | Path                                              |
```
Comment (vzotova) [inline README.md]:
```suggestion
| Function               | Collateral quantity                     | Path                                              |
```

File: README.md [hunk 4]
```diff
@@ -355,52 +349,52 @@ This ordered list is critical for gas-efficient redemption sequences and for the
...
+Nodes also remain sorted as the collateral:USD price varies, since price fluctuations change the collateral value of each Vault by the same proportion.
 
-Thus, nodes need only be re-inserted to the sorted list upon a Trove operation - when the owner adds or removes collateral or debt to their position.
+Thus, nodes need only be re-inserted to the sorted list upon a Vault operation - when the owner adds or removes collateral or debt to their position.
 
-### Flow of Ether in Threshold USD
+### Flow of Collateral in Threshold USD
 
-![Flow of Ether](images/ETH_flows.svg)
+![Flow of Collateral](images/ETH_flows.svg)
```
Comment (vzotova) [inline README.md]: just a remark: LQTY on picture

File: README.md [hunk 5]
```diff
@@ -410,25 +404,25 @@ Likewise, the StabilityPool holds the total accumulated ETH gains from liquidati
...
-| provideToSP            | depositor's accumulated ETH gain | StabilityPool -> msg.sender                       |
-| withdrawFromSP         | depositor's accumulated ETH gain | StabilityPool -> msg.sender                       |
-| withdrawETHGainToTrove | depositor's accumulated ETH gain | StabilityPool -> BorrowerOperations -> ActivePool |
+| provideToSP            | depositor's accumulated collateral gain | StabilityPool -> msg.sender                       |
+| withdrawFromSP         | depositor's accumulated collateral gain | StabilityPool -> msg.sender                       |
+| withdrawETHGainToTrove | depositor's accumulated collateral gain | StabilityPool -> BorrowerOperations -> ActivePool |
 
 ### Flow of thUSD tokens in Threshold USD
 
 ![Flow of thUSD](images/THUSD_flows.svg)
```
Comment (vzotova) [inline README.md:417]: I see here 404 when trying to check

**Review by vzotova — APPROVED** [review]

File: README.md (hunk 5 above)
Comment (evandrosaturnino) [reply README.md:417]: Addressed in https://github.com/Threshold-USD/dev/pull/106/commits/55db3c9409b0f9c70b259607697cd689ce06c20e

File: README.md (hunk 4 above)
Comment (evandrosaturnino) [reply README.md]: Addressed in https://github.com/Threshold-USD/dev/pull/106/commits/55db3c9409b0f9c70b259607697cd689ce06c20e

File: README.md (hunk 3 above)
Comment (evandrosaturnino) [reply README.md]: Addressed in https://github.com/Threshold-USD/dev/pull/106/commits/55db3c9409b0f9c70b259607697cd689ce06c20e

File: README.md (hunk 2 above)
Comment (evandrosaturnino) [reply README.md]: Addressed in https://github.com/Threshold-USD/dev/pull/106/commits/55db3c9409b0f9c70b259607697cd689ce06c20e

File: README.md (hunk 1 above)
Comment (evandrosaturnino) [reply README.md]: Addressed in https://github.com/Threshold-USD/dev/pull/106/commits/55db3c9409b0f9c70b259607697cd689ce06c20e

**Pull request merged**